image_width: 1200
image_height: 800
camera_name: /ovc/rgb/image_color
camera_matrix:
  rows: 3
  cols: 3
  data: [979.484523282704, 0, 667.7392808146, 0, 979.406640717951, 316.5795478571781, 0, 0, 1]
distortion_model: radtan
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.4016964684237056, 0.14838298250651594, 0.0005555170655668654, 0.000007075168918439364, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [979.484523282704, 0, 667.7392808146, 0, 0, 979.406640717951, 316.5795478571781, 0, 0, 0, 1, 0]
